src/Categories/Functors.lagda

```diff
@@ -0,0 +1,202 @@
+---
+layout: default
+title : "Categories.Functors module (The Agda Universal Algebra Library)"
+date : "2021-08-30"
+author: "agda-algebras development team"
+---
+
+### <a id="functors">Functors</a>
+
+This is the [Categories.Functors][] module of the [Agda Universal Algebra Library][].
+
+Recall, a *functor* `F` is a function that maps the objects and morphisms of one category 𝒞 to the objects and morphisms of a category 𝒟 in such a way that the following *functor laws* are satisfied:
+
+* ∀ f g, F(f ∘ g) = F(f) ∘ F(g)
+* ∀ A, F(id A) = id (F A)  (where id X denotes the identity map on X)
+
+
+#### <a id="polynomial-functors">Polynomial functors</a>
+
+The main reference for this section is [Modular Type-Safety Proofs in Agda](https://doi.org/10.1145/2428116.2428120) by Schwaab and Siek (PLPV '07).
+
+An important class of functors for our domain is the class of so called *polynomial functors*. These are functors that are built up using the constant, identity, sum, and product functors.  To be precise, the actions of the latter on objects are as follows: `∀ A B` (objects), `∀ F G` (functors),
+
+* `const B A = B`
+* `Id A = A`
+* `(F + G) A = F A + G A`
+* `(F × G) A = F A × G A`
+
+\begin{code}
+
+{-# OPTIONS --without-K --exact-split --safe #-}
+
+module Categories.Functors where
+
+-- Imports from Agda and the Agda Standard Library  ---------------------------------------
+open import Agda.Primitive using ( _⊔_ ; lsuc ; Level ) renaming ( Set to Type ; lzero to ℓ₀ )
+open import Data.Nat       using ( ℕ ; zero ; suc ; _>_ )
+open import Data.Sum.Base  using ( _⊎_ ) renaming ( inj₁ to inl ;  inj₂ to inr )
+open import Data.Product   using ( Σ-syntax ; _,_ ; _×_ )
+open import Data.Unit      using ( tt ) renaming ( ⊤ to ⊤₀ )
+open import Data.Unit.Polymorphic  using ( ⊤ )
+open import Relation.Binary.PropositionalEquality  using ( _≡_ ; refl ; _≢_ )
+open import Level
+
+private variable
+ α β : Level
+
+infixl 6 _⊕_
+infixr 7 _⊗_
+data Functor₀ : Type (lsuc ℓ₀) where
```

Comment:
Interesting... thank you for the comment.  I like when you point out these connections.  I'm very interested in connections between the concepts and language used in different areas (e.g. UA, Category Theory, Proof Theory, Type Theory). In fact, I'd like to build tools that could help to identify "redundancies" (i.e., when the same concept is presented in different language) not necessarily to eliminate them---since different terminology can be more/less useful in different contexts---but rather as a pedagogical device to help us better understand how different fields are related and to facilitate communication across disciplines.